### DISCUSSION TITLE
Possible to create trajectories of animals?

### DISCUSSION THREAD

**Discussion by KruemelmonsterK** [talmolab/sleap#1377] [Help!]
I want to track mutiple mice in their cage.
A common software for this is idtracker.ai, but it doent work in real-time.
Can I use SLEAP to track multile mice for a lonter time in real-time without mixing up their identitites?

**Comment by talmo** [comment]
Hi @KruemelmonsterK,

Yes and we recommend using the [Bonsai SLEAP extension](https://bonsai-rx.org/sleap/articles/intro.html) for real time applications.

Assuming your mice have distinct identities, you'll want to use a topdown supervised ID model in SLEAP. This requires assigning each animal to a track and training a supervised ID model (now available via the GUI).

Let us know if you have any questions!

**Comment by KruemelmonsterK** [comment]
Okay, to make things clear: I am not interested in pose estimation. I only want to create trajectories of multiple mice in real-time. It is important that no swapping of identities (e.g. after occlusion) occurs.

What do you mean with distinct identities? Its mice of the same breed, so they look quiet similar. Besides they are not marked. But its possible to mark the mice if necessary.

**Reply by roomrys** [reply]
Hi @KruemelmonsterK,

I'll try to step in here. The supervised ID models work best when the animals are visually distinct, so if you are able to have some sort of marking in a visible spot on the mice, then that would help in keeping tracks from switching. The non-ID models have shown to require a lot of proofreading to clean the tracks (keep them from switching) - so that isn't really an option for your case.

While SLEAP is set-up as a pose estimation software first, you can track however many points/nodes as you would like (forewarning that the bottom-up model expects at least 2 points/nodes).

Thanks,
Liezl
